### DISCUSSION TITLE
why isn't `conda update x` updating?

### DISCUSSION THREAD

**Discussion by M2ATrail** [conda-forge/miniforge#660] [Q&A] [answered]
curious why conda update duckdb isn't updating mine from 1.1.1 to 1.1.2 even though you seem to have added it to channel.
![image](https://github.com/user-attachments/assets/e1e34fbb-0c4c-498e-b5e8-005d6020e92e)

here's the verbose output
https://paste.pythondiscord.com/5CAW7AN4F2WVOTBLSAL7G3IISQ

interesting enough, `conda update --all` recognizes it.
![image](https://github.com/user-attachments/assets/3fc03533-b0a0-4ef8-9950-86ff28280d79)

**Comment by hmaarrfk** [comment]
i have stopped trusting the update command myself.

I mostly say
```
conda install "package>=the.version.i.want"
```

**Reply by M2ATrail** [reply]
Well, I'm just checking if I'm stupid or if I should move this to issue.

**Reply by hmaarrfk** [reply, marked as answer]
An issue with conda-feedstock. Potentially upstream to conda/conda 

But mini forge is working ;)

**Reply by M2ATrail** [reply]
oh whoops, I wrote conda and history sent me here, my bad.
https://github.com/conda/conda/issues/14324
